molecule/default/converge.yml

```diff
@@ -3,8 +3,11 @@
   hosts: all
   tasks:
     - name: Include ansible-role-xfce-cool
-      ansible.builtin.include_role:
+       # We do prepend the name of the role to the role variables, but
```

Comment:
Fix indentation:
```suggestion
      # We do prepend the name of the role to the role variables, but
```

Reply:
@jsf9k The indentation matches the code block it actually applies to, which is the same way it's done in https://github.com/cisagov/ansible-role-python/blob/711e0de8cb026663908aef2e5bbe265ecf3e6602/molecule/python2/converge.yml#L6

Reply:
My bad, just noticed there's an extra indent on that top line, fixing now

Reply:
Fixed in commit d367078.

Reply:
See https://github.com/cisagov/ansible-role-xfce-cool/commit/d367078681dd6240a3a7b11ad00cfbf83b684f59